XSConsoleTask.py

```diff
@@ -53,11 +56,7 @@ def Status(self):
         return status
 
     def Result(self):
-        if self.Completed():
-            result = self.completionStatus
-        else:
-            result= self.session.xenapi.task.get_status(self.hotOpaqueRef.OpaqueRef())
```

Comment:
Previously this returned the status which seems wrong. Was that used anywhere or was it just a latent bug?

Reply:
I believe this is a bug, and I checked the function was not used.